src/API/index.js

```diff
@@ -13,3 +13,21 @@ export const signinAPI = {
     return axios.post(`${baseURL}api/signin`, userData);
   }
 }
+
+export const createBoardAPI = {
+  createBoard(boardData) {
+    return axios.post(`${baseURL}api/boards/create`, boardData);
+  }
+}
+
+export const getUserBoardsAPI = {
+  requestUserBoards() {
+    return axios.get(`${baseURL}api/boards/get`);
+  }
+}
+
+export const verifyTokenAPI = {
+  verify() {
+    return axios.get(`${baseURL}api/verify`);
+  }
+}
```

Comment:
Doesn't make sense to wrap this API calls into objects, you have only 1 API everywhere